student-store-ui/src/components/Contact/Contact.css

```diff
@@ -0,0 +1,80 @@
+
```

Comment:
as using css files -> naming the tags with a some type of prefix helps to identify its usage / not have the css overwriting in unintentional areas if importing multiple files